Sources/SmokeDynamoDB/DynamoDBCompositePrimaryKeyTable.swift

```diff
@@ -150,7 +150,8 @@ public protocol DynamoDBCompositePrimaryKeyTable {
        the query.
      */
     func query<ReturnedType: PolymorphicOperationReturnType>(forPartitionKey partitionKey: String,
-                                                             sortKeyCondition: AttributeCondition?)
+                                                             sortKeyCondition: AttributeCondition?,
+                                                             consistentRead: Bool?)
```

Comment:
In my opinion, making this an optional boolean is just confusing as nil and false are equivalent.